dvc/repo/get_url.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import dvc.output as output
+import dvc.dependency as dependency
+
+from dvc.utils.compat import urlparse
+
+
+@staticmethod
+def get_url(url, out=None):
+    out = out or os.path.basename(urlparse(url).path)
+
+    if os.path.exists(url):
+        src = os.path.abspath(url)
+        out = os.path.abspath(out)
+
+    dep, = dependency.loads_from(None, [src])
```

Comment:
will it support remote:// protocol? it needs to read a global config for that, right?

Reply:
It won't, at least for now. `get-url` doesn't care about Repo's at all. I would keep it this way untill someone asks for this feature.